frontend/public/components/nav/items.tsx

```diff
@@ -29,7 +29,13 @@ export const stripNS = (href) => {
 
 export const ExternalLink = ({ href, name }) => (
   <NavItem isActive={false}>
-    <a className="pf-c-nav__link" href={href} target="_blank" rel="noopener noreferrer">
+    <a
+      className="pf-c-nav__link"
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      data-cy="nav"
```

Comment:
This should be `data-test` per [our best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#migrating-protractor-tests-to-cypress)